### ISSUE TITLE
APT Repo Broken

### ISSUE THREAD

**Issue by Ruok2bu** [pkoutoupis/rapiddisk#30]
Followed instructions in "Ubuntu LTS 16.04" section of wiki.  Updating apt results in this error:

`gn:11 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main all Packages
Ign:15 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en_CA
Ign:16 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en
Ign:11 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main all Packages
Ign:15 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en_CA
Ign:16 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en
Ign:11 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main all Packages
Get:13 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages [491 B]
Ign:13 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages
Ign:15 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en_CA
Ign:16 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en
Ign:11 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main all Packages
Ign:13 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages
Ign:15 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en_CA
Ign:16 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en
Ign:11 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main all Packages
Ign:13 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages
Ign:13 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages
Ign:15 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en_CA
Ign:16 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en
Ign:11 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main all Packages
Ign:13 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages
Ign:15 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en_CA
Ign:16 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en
Err:13 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages
  lzma_read: Read error (7)
Fetched 491 B in 6s (77 B/s)
Reading package lists... Done
E: Failed to fetch http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604/dists/stable/main/binary-amd64/Packages  lzma_read: Read error (7)
E: Some index files failed to download. They have been ignored, or old ones used instead.
`

P.S. My OS is Debian 9.6

**Comment by pkoutoupis** [comment]
Such an error would indicate corruption of a file, although it seems to work fine on an Ubuntu 16.04 installation. In theory, it should work since both are based off of Stretch.

```
root@ubuntu1604-test:/home/petros# aptitude update
Hit http://us.archive.ubuntu.com/ubuntu xenial InRelease
Hit http://security.ubuntu.com/ubuntu xenial-security InRelease                 
Hit http://us.archive.ubuntu.com/ubuntu xenial-updates InRelease                
Hit http://us.archive.ubuntu.com/ubuntu xenial-backports InRelease    
Ign http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable InRelease
Ign http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable Release
Get: 1 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages [491 B]
Get: 2 http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main amd64 Packages [491 B]
Ign http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main all Packages
Ign http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en_US
Ign http://downloads.rapiddisk.org/pub/releases/5/ubuntu1604 stable/main Translation-en
Fetched 491 B in 2s (165 B/s)
```

I will install Debian 9.6 and try again.

**Comment by pkoutoupis** [comment]
OK. I was able to reproduce on the same Debian release. Couple of thoughts here...

Debian was never officially supported but I am surprised that it will generate this issue.

Sometime ago, maybe version 5.1, I stopped building the prebuilt binaries and uploading them to the repos (for all platforms). It just became way too much work. I just updated the Wiki to show the section and support for this to be deprecated (https://github.com/pkoutoupis/rapiddisk/issues/24#issuecomment-435589966).

To be honest though, you would be getting the latest version by building from the latest source release.
